db/migrate/20200521150213_create_storage_services.rb

```diff
@@ -0,0 +1,14 @@
+class CreateStorageServices < ActiveRecord::Migration[5.2]
+  def change
+    create_table :storage_services do |t|
+      t.string :name
+      t.string :description
+      t.string :uuid
```

Comment:
@agrare Based on your comment on the pyhsical_storages table, should this, then, be uid_ems instead of uuid?

Reply:
:+1: yeah